hexrd/findorientations.py

```diff
@@ -388,56 +388,48 @@ def load_eta_ome_maps(cfg, pd, image_series, hkls=None, clean=False):
 
     Parameters
     ----------
-    cfg : TYPE
-        DESCRIPTION.
-    pd : TYPE
-        DESCRIPTION.
-    image_series : TYPE
-        DESCRIPTION.
-    hkls : TYPE, optional
-        DESCRIPTION. The default is None.
-    clean : TYPE, optional
-        DESCRIPTION. The default is False.
+    cfg: Config instance
+        root config file for this problem
+    pd: PlaneData instance
+        crystallographic properties
+    image_series: ImageSeries instance
+        stack of images
+    hkls: list, default = None
+        list of HKLs used in the eta-omega maps.
```

Comment:
I'll update these docs. First, it will try to load the file if they have already been computed. If not, it will use the HKJs in the config file unless the HKLs are specifically passed here; then the passed HKLs override.

Reply:
Okay, thanks Don!